src/main/java/org/breedinginsight/brapi/v2/dao/BrAPIListDAO.java

```diff
@@ -196,4 +214,19 @@ public List<BrAPIListSummary> createBrAPILists(List<BrAPIListNewRequest> brapiLi
 
         throw new ApiException("No response after creating list");
     }
+
+    public HttpResponse<String> deleteBrAPIList(String listDbId, UUID programId, boolean hardDelete) throws ApiException {
+        // TODO: Switch to using the ListsApi from the BrAPI client library once the delete endpoints are merged into it
```

Comment:
Is there a card the todos are associated with?

Reply:
Yep, I updated the comment to include the card number.